### DISCUSSION TITLE
[5.x] How to see updates to matrix blocks or blocks in CKeditor without affecting the online version of the content?

### DISCUSSION THREAD

**Discussion by scoch5** [craftcms/cms#14727] [Q&A] [answered]
When I modify an entry with the preview open and I am using a matrix block or a CKeditor block with entry types inside to modify these blocks, an overlay panel opens and only once this new Entry is updated then I see the updated preview .
The problem is that when it is saved, the public site also immediately undergoes this change.
To get around the problem I tried to create a new draft and apply the changes there, unfortunately the problem occurs the same way immediately updating the public site.

Basically I haven't found a way to see updates to matrix blocks or blocks in CKeditor without affecting the online version of the content.

**Comment by thupsi** [comment]
This behavior has been discussed in Discord (also see [#14603](https://github.com/craftcms/cms/discussions/14603)) and Brandon has started work on an updated workflow for nested entries. Basically, the default behavior will be to save changes to slideouts as drafts, but also give the user an option to save immediately. According to the public roadmap this is slated for 5.1.

**Comment by thupsi** [comment]
In the meantime you can assing a URI pattern to a Matrix field, create a draft for the nested block and preview it in that URL. **You can't do that for CKEditor fields though**, maybe a good FR idea.

A related FR is [#14298](https://github.com/craftcms/cms/discussions/14298), which would help a lot if implemented.

**Comment by brandonkelly** [comment, marked as answer]
Made some improvements to working with nested entries (and addresses) for Craft 5.5: #16002 🎉 CKEditor will get updated with the same changes as well.

**Reply by brandonkelly** [reply]
Craft 5.5.0 and CKEditor 4.4.0 are out with those changes 🎉
